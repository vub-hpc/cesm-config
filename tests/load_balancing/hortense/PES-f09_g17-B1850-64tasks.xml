<?xml version="1.0"?>
<config_pes>
  <grid name="any">
    <mach name="any">
      <pes compset="any" pesize="4nS">
        <comment>Sequential</comment>
        <ntasks>
          <ntasks_atm>64</ntasks_atm>
          <ntasks_lnd>64</ntasks_lnd>
          <ntasks_rof>64</ntasks_rof>
          <ntasks_ice>64</ntasks_ice>
          <ntasks_ocn>64</ntasks_ocn>
          <ntasks_glc>64</ntasks_glc>
          <ntasks_wav>64</ntasks_wav>
          <ntasks_cpl>64</ntasks_cpl>
        </ntasks>
        <nthrds>
          <nthrds_atm>1</nthrds_atm>
          <nthrds_lnd>1</nthrds_lnd>
          <nthrds_rof>1</nthrds_rof>
          <nthrds_ice>1</nthrds_ice>
          <nthrds_ocn>1</nthrds_ocn>
          <nthrds_glc>1</nthrds_glc>
          <nthrds_wav>1</nthrds_wav>
          <nthrds_cpl>1</nthrds_cpl>
        </nthrds>
        <rootpe>
          <rootpe_atm>0</rootpe_atm>
          <rootpe_lnd>0</rootpe_lnd>
          <rootpe_rof>0</rootpe_rof>
          <rootpe_ice>0</rootpe_ice>
          <rootpe_ocn>0</rootpe_ocn>
          <rootpe_glc>0</rootpe_glc>
          <rootpe_wav>0</rootpe_wav>
          <rootpe_cpl>0</rootpe_cpl>
        </rootpe>
      </pes>
    </mach>
  </grid>
  <grid name="any">
    <mach name="any">
      <pes compset="any" pesize="4nO25">
        <comment>Concurrent OCN at 25 percent</comment>
        <ntasks>
          <ntasks_atm>48</ntasks_atm>
          <ntasks_lnd>48</ntasks_lnd>
          <ntasks_rof>48</ntasks_rof>
          <ntasks_ice>48</ntasks_ice>
          <ntasks_ocn>16</ntasks_ocn>
          <ntasks_glc>48</ntasks_glc>
          <ntasks_wav>48</ntasks_wav>
          <ntasks_cpl>48</ntasks_cpl>
        </ntasks>
        <nthrds>
          <nthrds_atm>1</nthrds_atm>
          <nthrds_lnd>1</nthrds_lnd>
          <nthrds_rof>1</nthrds_rof>
          <nthrds_ice>1</nthrds_ice>
          <nthrds_ocn>1</nthrds_ocn>
          <nthrds_glc>1</nthrds_glc>
          <nthrds_wav>1</nthrds_wav>
          <nthrds_cpl>1</nthrds_cpl>
        </nthrds>
        <rootpe>
          <rootpe_atm>0</rootpe_atm>
          <rootpe_lnd>0</rootpe_lnd>
          <rootpe_rof>0</rootpe_rof>
          <rootpe_ice>0</rootpe_ice>
          <rootpe_ocn>48</rootpe_ocn>
          <rootpe_glc>0</rootpe_glc>
          <rootpe_wav>0</rootpe_wav>
          <rootpe_cpl>0</rootpe_cpl>
        </rootpe>
      </pes>
    </mach>
  </grid>
  <grid name="any">
    <mach name="any">
      <pes compset="any" pesize="4nO50">
        <comment>Concurrent OCN at 50 percent</comment>
        <ntasks>
          <ntasks_atm>32</ntasks_atm>
          <ntasks_lnd>32</ntasks_lnd>
          <ntasks_rof>32</ntasks_rof>
          <ntasks_ice>32</ntasks_ice>
          <ntasks_ocn>32</ntasks_ocn>
          <ntasks_glc>32</ntasks_glc>
          <ntasks_wav>32</ntasks_wav>
          <ntasks_cpl>32</ntasks_cpl>
        </ntasks>
        <nthrds>
          <nthrds_atm>1</nthrds_atm>
          <nthrds_lnd>1</nthrds_lnd>
          <nthrds_rof>1</nthrds_rof>
          <nthrds_ice>1</nthrds_ice>
          <nthrds_ocn>1</nthrds_ocn>
          <nthrds_glc>1</nthrds_glc>
          <nthrds_wav>1</nthrds_wav>
          <nthrds_cpl>1</nthrds_cpl>
        </nthrds>
        <rootpe>
          <rootpe_atm>0</rootpe_atm>
          <rootpe_lnd>0</rootpe_lnd>
          <rootpe_rof>0</rootpe_rof>
          <rootpe_ice>0</rootpe_ice>
          <rootpe_ocn>32</rootpe_ocn>
          <rootpe_glc>0</rootpe_glc>
          <rootpe_wav>0</rootpe_wav>
          <rootpe_cpl>0</rootpe_cpl>
        </rootpe>
      </pes>
    </mach>
  </grid>
  <grid name="any">
    <mach name="any">
      <pes compset="any" pesize="4nI50L50">
        <comment>Concurrent ICE/LND at 50/50 in sequence with ATM</comment>
        <ntasks>
          <ntasks_atm>64</ntasks_atm>
          <ntasks_lnd>32</ntasks_lnd>
          <ntasks_rof>64</ntasks_rof>
          <ntasks_ice>32</ntasks_ice>
          <ntasks_ocn>64</ntasks_ocn>
          <ntasks_glc>64</ntasks_glc>
          <ntasks_wav>64</ntasks_wav>
          <ntasks_cpl>64</ntasks_cpl>
        </ntasks>
        <nthrds>
          <nthrds_atm>1</nthrds_atm>
          <nthrds_lnd>1</nthrds_lnd>
          <nthrds_rof>1</nthrds_rof>
          <nthrds_ice>1</nthrds_ice>
          <nthrds_ocn>1</nthrds_ocn>
          <nthrds_glc>1</nthrds_glc>
          <nthrds_wav>1</nthrds_wav>
          <nthrds_cpl>1</nthrds_cpl>
        </nthrds>
        <rootpe>
          <rootpe_atm>0</rootpe_atm>
          <rootpe_lnd>0</rootpe_lnd>
          <rootpe_rof>0</rootpe_rof>
          <rootpe_ice>32</rootpe_ice>
          <rootpe_ocn>0</rootpe_ocn>
          <rootpe_glc>0</rootpe_glc>
          <rootpe_wav>0</rootpe_wav>
          <rootpe_cpl>0</rootpe_cpl>
        </rootpe>
      </pes>
    </mach>
  </grid>
  <grid name="any">
    <mach name="any">
      <pes compset="any" pesize="4nO25I50L50">
        <comment>Concurrent OCN at 25 percent with concurrent ICE/LND at 50/50 in sequence with ATM</comment>
        <ntasks>
          <ntasks_atm>48</ntasks_atm>
          <ntasks_lnd>24</ntasks_lnd>
          <ntasks_rof>48</ntasks_rof>
          <ntasks_ice>24</ntasks_ice>
          <ntasks_ocn>16</ntasks_ocn>
          <ntasks_glc>48</ntasks_glc>
          <ntasks_wav>48</ntasks_wav>
          <ntasks_cpl>48</ntasks_cpl>
        </ntasks>
        <nthrds>
          <nthrds_atm>1</nthrds_atm>
          <nthrds_lnd>1</nthrds_lnd>
          <nthrds_rof>1</nthrds_rof>
          <nthrds_ice>1</nthrds_ice>
          <nthrds_ocn>1</nthrds_ocn>
          <nthrds_glc>1</nthrds_glc>
          <nthrds_wav>1</nthrds_wav>
          <nthrds_cpl>1</nthrds_cpl>
        </nthrds>
        <rootpe>
          <rootpe_atm>0</rootpe_atm>
          <rootpe_lnd>0</rootpe_lnd>
          <rootpe_rof>0</rootpe_rof>
          <rootpe_ice>24</rootpe_ice>
          <rootpe_ocn>48</rootpe_ocn>
          <rootpe_glc>0</rootpe_glc>
          <rootpe_wav>0</rootpe_wav>
          <rootpe_cpl>0</rootpe_cpl>
        </rootpe>
      </pes>
    </mach>
  </grid>
  <grid name="any">
    <mach name="any">
      <pes compset="any" pesize="4nO25I75L25">
        <comment>Concurrent OCN at 25 percent with concurrent ICE/LND at 75/25 in sequence with ATM</comment>
        <ntasks>
          <ntasks_atm>48</ntasks_atm>
          <ntasks_lnd>16</ntasks_lnd>
          <ntasks_rof>48</ntasks_rof>
          <ntasks_ice>32</ntasks_ice>
          <ntasks_ocn>16</ntasks_ocn>
          <ntasks_glc>48</ntasks_glc>
          <ntasks_wav>48</ntasks_wav>
          <ntasks_cpl>48</ntasks_cpl>
        </ntasks>
        <nthrds>
          <nthrds_atm>1</nthrds_atm>
          <nthrds_lnd>1</nthrds_lnd>
          <nthrds_rof>1</nthrds_rof>
          <nthrds_ice>1</nthrds_ice>
          <nthrds_ocn>1</nthrds_ocn>
          <nthrds_glc>1</nthrds_glc>
          <nthrds_wav>1</nthrds_wav>
          <nthrds_cpl>1</nthrds_cpl>
        </nthrds>
        <rootpe>
          <rootpe_atm>0</rootpe_atm>
          <rootpe_lnd>0</rootpe_lnd>
          <rootpe_rof>0</rootpe_rof>
          <rootpe_ice>16</rootpe_ice>
          <rootpe_ocn>48</rootpe_ocn>
          <rootpe_glc>0</rootpe_glc>
          <rootpe_wav>0</rootpe_wav>
          <rootpe_cpl>0</rootpe_cpl>
        </rootpe>
      </pes>
    </mach>
  </grid>
  <grid name="any">
    <mach name="any">
      <pes compset="any" pesize="4nO25I25L75">
        <comment>Concurrent OCN at 25 percent with concurrent ICE/LND at 25/75 in sequence with ATM</comment>
        <ntasks>
          <ntasks_atm>48</ntasks_atm>
          <ntasks_lnd>32</ntasks_lnd>
          <ntasks_rof>48</ntasks_rof>
          <ntasks_ice>16</ntasks_ice>
          <ntasks_ocn>16</ntasks_ocn>
          <ntasks_glc>48</ntasks_glc>
          <ntasks_wav>48</ntasks_wav>
          <ntasks_cpl>48</ntasks_cpl>
        </ntasks>
        <nthrds>
          <nthrds_atm>1</nthrds_atm>
          <nthrds_lnd>1</nthrds_lnd>
          <nthrds_rof>1</nthrds_rof>
          <nthrds_ice>1</nthrds_ice>
          <nthrds_ocn>1</nthrds_ocn>
          <nthrds_glc>1</nthrds_glc>
          <nthrds_wav>1</nthrds_wav>
          <nthrds_cpl>1</nthrds_cpl>
        </nthrds>
        <rootpe>
          <rootpe_atm>0</rootpe_atm>
          <rootpe_lnd>0</rootpe_lnd>
          <rootpe_rof>0</rootpe_rof>
          <rootpe_ice>32</rootpe_ice>
          <rootpe_ocn>48</rootpe_ocn>
          <rootpe_glc>0</rootpe_glc>
          <rootpe_wav>0</rootpe_wav>
          <rootpe_cpl>0</rootpe_cpl>
        </rootpe>
      </pes>
    </mach>
  </grid>
  <grid name="any">
    <mach name="any">
      <pes compset="any" pesize="4nO50I50L50">
        <comment>Concurrent OCN at 50 percent with concurrent ICE/LND at 50/50 in sequence with ATM</comment>
        <ntasks>
          <ntasks_atm>32</ntasks_atm>
          <ntasks_lnd>16</ntasks_lnd>
          <ntasks_rof>32</ntasks_rof>
          <ntasks_ice>16</ntasks_ice>
          <ntasks_ocn>32</ntasks_ocn>
          <ntasks_glc>32</ntasks_glc>
          <ntasks_wav>32</ntasks_wav>
          <ntasks_cpl>32</ntasks_cpl>
        </ntasks>
        <nthrds>
          <nthrds_atm>1</nthrds_atm>
          <nthrds_lnd>1</nthrds_lnd>
          <nthrds_rof>1</nthrds_rof>
          <nthrds_ice>1</nthrds_ice>
          <nthrds_ocn>1</nthrds_ocn>
          <nthrds_glc>1</nthrds_glc>
          <nthrds_wav>1</nthrds_wav>
          <nthrds_cpl>1</nthrds_cpl>
        </nthrds>
        <rootpe>
          <rootpe_atm>0</rootpe_atm>
          <rootpe_lnd>0</rootpe_lnd>
          <rootpe_rof>0</rootpe_rof>
          <rootpe_ice>16</rootpe_ice>
          <rootpe_ocn>32</rootpe_ocn>
          <rootpe_glc>0</rootpe_glc>
          <rootpe_wav>0</rootpe_wav>
          <rootpe_cpl>0</rootpe_cpl>
        </rootpe>
      </pes>
    </mach>
  </grid>
</config_pes>
