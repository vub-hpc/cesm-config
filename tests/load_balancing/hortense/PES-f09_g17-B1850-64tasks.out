found /dodrio/scratch/users/vsc10122/cesm/output/PFS_I7.f09_g17.B1850.hortense_gnu.lbt/timing
found /dodrio/scratch/users/vsc10122/cesm/output/PFS_I2.f09_g17.B1850.hortense_gnu.lbt/timing
found /dodrio/scratch/users/vsc10122/cesm/output/PFS_I4.f09_g17.B1850.hortense_gnu.lbt/timing
found /dodrio/scratch/users/vsc10122/cesm/output/PFS_I0.f09_g17.B1850.hortense_gnu.lbt/timing
found /dodrio/scratch/users/vsc10122/cesm/output/PFS_I5.f09_g17.B1850.hortense_gnu.lbt/timing
found /dodrio/scratch/users/vsc10122/cesm/output/PFS_I3.f09_g17.B1850.hortense_gnu.lbt/timing
found /dodrio/scratch/users/vsc10122/cesm/output/PFS_I1.f09_g17.B1850.hortense_gnu.lbt/timing
found /dodrio/scratch/users/vsc10122/cesm/output/PFS_I6.f09_g17.B1850.hortense_gnu.lbt/timing
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I0.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing.PFS_I0.f09_g17.B1850.hortense_gnu.lbt.1835910.220922-151544
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I0.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing_stats.1835910.220922-151544
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I1.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing_stats.1835919.220922-151757
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I1.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing.PFS_I1.f09_g17.B1850.hortense_gnu.lbt.1835919.220922-151757
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I2.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing.PFS_I2.f09_g17.B1850.hortense_gnu.lbt.1835922.220922-152034
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I2.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing_stats.1835922.220922-152034
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I3.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing_stats.1835926.220922-152550
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I3.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing.PFS_I3.f09_g17.B1850.hortense_gnu.lbt.1835926.220922-152550
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I4.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing.PFS_I4.f09_g17.B1850.hortense_gnu.lbt.1835977.220922-153125
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I4.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing_stats.1835977.220922-153125
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I5.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing_stats.1835979.220922-153125
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I5.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing.PFS_I5.f09_g17.B1850.hortense_gnu.lbt.1835979.220922-153125
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I6.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing.PFS_I6.f09_g17.B1850.hortense_gnu.lbt.1835981.220922-153432
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I6.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing_stats.1835981.220922-153432
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I7.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing_stats.1835982.220922-153733
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I7.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing.PFS_I7.f09_g17.B1850.hortense_gnu.lbt.1835982.220922-153733
Using layout  = IceLndWavAtmOcn
GLPSOL--GLPK LP/MIP Solver 5.0
Parameter(s) specified in the command line:
 --cpxlp /tmp/4da4dcdcd48c418fbfb7ab13c3eea3e8-pulp.lp -o /tmp/4da4dcdcd48c418fbfb7ab13c3eea3e8-pulp.sol
Reading problem data from '/tmp/4da4dcdcd48c418fbfb7ab13c3eea3e8-pulp.lp'...
29 rows, 17 columns, 61 non-zeros
10 integer variables, none of which are binary
56 lines were read
GLPK Integer Optimizer 5.0
29 rows, 17 columns, 61 non-zeros
10 integer variables, none of which are binary
Preprocessing...
29 rows, 17 columns, 61 non-zeros
10 integer variables, none of which are binary
Scaling...
 A: min|aij| =  1.000e+00  max|aij| =  7.720e+03  ratio =  7.720e+03
GM: min|aij| =  2.360e-01  max|aij| =  4.238e+00  ratio =  1.796e+01
EQ: min|aij| =  5.568e-02  max|aij| =  1.000e+00  ratio =  1.796e+01
2N: min|aij| =  3.951e-02  max|aij| =  1.395e+00  ratio =  3.531e+01
Constructing initial basis...
Size of triangular part is 29
Solving LP relaxation...
GLPK Simplex Optimizer 5.0
29 rows, 17 columns, 61 non-zeros
      0: obj =   5.188328500e+03 inf =   8.788e+02 (13)
     20: obj =   8.533950878e+03 inf =   8.358e-15 (0)
*    24: obj =   8.001303603e+03 inf =   0.000e+00 (0)
OPTIMAL LP SOLUTION FOUND
Integer optimization begins...
Long-step dual simplex will be used
+    24: mip =     not found yet >=              -inf        (1; 0)
+    30: >>>>>   8.627757000e+03 >=   8.627757000e+03 < 0.1% (4; 0)
+    30: mip =   8.627757000e+03 >=     tree is empty   0.0% (0; 7)
INTEGER OPTIMAL SOLUTION FOUND
Time used:   0.0 secs
Memory used: 0.1 Mb (91553 bytes)
Writing MIP solution to '/tmp/4da4dcdcd48c418fbfb7ab13c3eea3e8-pulp.sol'...
Solving Mixed Integer Linear Program using PuLP interface to COIN-CBC
GLPSOL--GLPK LP/MIP Solver 5.0
Parameter(s) specified in the command line:
 --cpxlp /tmp/52e927096a8841b3bf1a6957a9522383-pulp.lp -o /tmp/52e927096a8841b3bf1a6957a9522383-pulp.sol
Reading problem data from '/tmp/52e927096a8841b3bf1a6957a9522383-pulp.lp'...
29 rows, 17 columns, 61 non-zeros
10 integer variables, none of which are binary
56 lines were read
GLPK Integer Optimizer 5.0
29 rows, 17 columns, 61 non-zeros
10 integer variables, none of which are binary
Preprocessing...
29 rows, 17 columns, 61 non-zeros
10 integer variables, none of which are binary
Scaling...
 A: min|aij| =  1.000e+00  max|aij| =  7.720e+03  ratio =  7.720e+03
GM: min|aij| =  2.360e-01  max|aij| =  4.238e+00  ratio =  1.796e+01
EQ: min|aij| =  5.568e-02  max|aij| =  1.000e+00  ratio =  1.796e+01
2N: min|aij| =  3.951e-02  max|aij| =  1.395e+00  ratio =  3.531e+01
Constructing initial basis...
Size of triangular part is 29
Solving LP relaxation...
GLPK Simplex Optimizer 5.0
29 rows, 17 columns, 61 non-zeros
      0: obj =   5.188328500e+03 inf =   8.788e+02 (13)
     20: obj =   8.533950878e+03 inf =   8.358e-15 (0)
*    24: obj =   8.001303603e+03 inf =   0.000e+00 (0)
OPTIMAL LP SOLUTION FOUND
Integer optimization begins...
Long-step dual simplex will be used
+    24: mip =     not found yet >=              -inf        (1; 0)
+    30: >>>>>   8.627757000e+03 >=   8.627757000e+03 < 0.1% (4; 0)
+    30: mip =   8.627757000e+03 >=     tree is empty   0.0% (0; 7)
INTEGER OPTIMAL SOLUTION FOUND
Time used:   0.0 secs
Memory used: 0.1 Mb (91553 bytes)
Writing MIP solution to '/tmp/52e927096a8841b3bf1a6957a9522383-pulp.sol'...
PuLP solver status: Solved
COST_ATM = 6252.160000
COST_ICE = 2375.600000
COST_LND = 1820.580000
COST_OCN = 8627.760000
COST_TOTAL = 8627.760000
COST_WAV = 2375.600000
NBLOCKS_ATM = 6
NBLOCKS_ICE = 2
NBLOCKS_LND = 2
NBLOCKS_OCN = 2
NBLOCKS_WAV = 2
NTASKS_ATM = 48
NTASKS_ICE = 16
NTASKS_LND = 16
NTASKS_OCN = 16
NTASKS_TOTAL = 64
NTASKS_WAV = 16
